app/controllers/concerns/error_handling.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ErrorHandling
+  extend ActiveSupport::Concern
+
+  protected
+
+  def render_bad_request(resource)
+    render_bad_request_json(resource)
```

Comment:
where is this method defined?

Reply:
That was a left over, I removed it now :)